vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -146,48 +142,48 @@ Here is an example:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: YOUR-TENANTS-CONFIGMAP
+  name: YOUR-TENANT1-CONFIGMAP
 data:
   ENTANDO_TENANTS: >-
         [
             {
                 "tenantCode": "YOUR-TENANT1-CODE", # default subdomain name 
-                "fqdns": "YOUR-TENANT1-CODE.YOUR-DOMAIN" # value string and comma separated domains
+                "fqdns": "YOUR-TENANT1-CODE.YOUR-HOSTNAME" # value string and comma separated domains
                 "kcEnabled": true,
-                "kcAuthUrl": "http://YOUR-DOMAIN/auth",
+                "kcAuthUrl": "http://YOUR-HOSTNAME/auth",
                 "kcRealm": "YOUR-TENANT1-REALM",
                 "kcClientId": "quickstart",
-                "kcClientSecret": "YOUR-TENANT1-SECRET",
+                "kcClientSecret": "YOUR-TENANT1-KC-SECRET",
                 "kcPublicClientId": "entando-web",
                 "kcSecureUris": "",
                 "kcDefaultAuthorizations": "",
                 "dbDriverClassName": "org.postgresql.Driver",
                 "dbUrl": "YOUR-POSTGRESQL-TENANT1-URL",
                 "dbUsername": "postgres",
                 "dbPassword": "e7d60efa865c4510",
-                "cdsPublicUrl": "http://cds.YOUR-DOMAIN/YOUR-TENANT1-CODE/",
-                "cdsPrivateUrl": "http://YOUR-TENANT1-CODE-cds:8080/",
+                "cdsPublicUrl": "http://cds.YOUR-HOSTNAME/YOUR-TENANT1-CODE/",
+                "cdsPrivateUrl": "http://mt720-cds-tenant1-service.test-mt-720.svc.cluster.local:8080/",
                 "cdsPath": "api/v1",
                 "solrAddress": "http://YOUR-SOLR-URL",
                 "solrCore": "TENANT1-CORE-NAME"
             },
             {
                 "tenantCode": "YOUR-TENANT2-CODE",
-                "fqdns": "www.YOUR-TENANT2-CODE.com,YOUR-TENANT2-CODE.YOUR-DOMAIN.com,news.YOUR-DOMAIN.com"
+                "fqdns": "www.YOUR-TENANT2-CODE.com,YOUR-TENANT2-CODE.YOUR-HOSTNAME.com,news.YOUR-HOSTNAME.com"
                 "kcEnabled": true,
-                "kcAuthUrl": "https://YOUR-DOMAIN/auth",
+                "kcAuthUrl": "https://YOUR-HOSTNAME/auth",
                 "kcRealm": "YOUR-TENANT2-REALM",
                 "kcClientId": "quickstart",
-                "kcClientSecret": "YOUR-TENANT2-SECRET",
+                "kcClientSecret": "YOUR-TENANT2-KC-SECRET",
                 "kcPublicClientId": "entando-web",
                 "kcSecureUris": "",
                 "kcDefaultAuthorizations": "",
                 "dbDriverClassName": "org.postgresql.Driver",
                 "dbUrl": "YOUR-POSTGRESQL-TENANT2-URL",
                 "dbUsername": "postgres",
                 "dbPassword": "xxx",
-                "cdsPublicUrl": "http://cds.YOUR-DOMAIN/YOUR-TENANT2-CODE/",,
-                "cdsPrivateUrl": "http://mt720-cds-YOUR-TENANT2-CODE-service.test-mt-720.svc.cluster.local:8080/",
+                "cdsPublicUrl": "http://cds.YOUR-HOSTNAME/YOUR-TENANT2-CODE/",,
```

Comment:
@jyunmitch cdsPrivateUrl **Must** follow the naming convention for K8S internal name dns resolutions, i.e:
http://<service-name>.<namespace>.svc.cluster.local:<service-port>[which is 8080 for the CDS internal service]

While for the public cds url, it will depends more on the type of DNS records you have at your disposal:
- if you have a wildcard dns record you can use: http|https://<whatever-you-want>.your-domain/<whatever-name-you-gave-to-the-tenant>/
- if you have a static DNS entry you need to use that entry: http|https://your-fqdn/<whatever-name-you-gave-to-the-tenant>

For the solr Address you should apply the same rule as the cdsPrivateUrl: http://<service-name>.<namespace>.svc.cluster.local:<solr-service-port>/solr

Reply:
@pietrangelo the public urls should probably be https by default, correct? I would only specify that here since it's the recommended configuration. The internal cluster address I except to just be http.

Reply:
@pietrangelo, for Solr, there is a core for secondary tenant1 but not a service. Is the `service-name` in the solrAddress  supposed to refer to the primary Solr service?